src/installer/pkg/projects/Microsoft.DotNet.ILCompiler/Directory.Build.props

```
@@ -0,0 +1,10 @@
+<Project>
+
+  <!-- Must set this before importing the file below -->
+  <PropertyGroup>
+    <RIDPropsFile>$(MSBuildThisFileDirectory)ILCompiler-appRIDs.props</RIDPropsFile>
```

```suggestion
    <RIDPropsFile>$(MSBuildThisFileDirectory)ILCompilerRIDs.props</RIDPropsFile>
```
"app" does not make sense to me here.